### DISCUSSION TITLE
502 Bad Gateway

### DISCUSSION THREAD

**Discussion by AmaralSza** [cleitonme/izing.instalador#4] [Q&A]
Boa noite, fiz um teste usando o instalador automático, funcionou, porém depois de atualizar usando a própria ferramenta de instalação na opção de atualizar, a pagina retorna 502 Bad Gateway, o que pode ser?

**Comment by cleitonme** [comment]
Sistema chegou funciona? E parou depois atualizar? Preciso mais detalhes fica muito vago o que pode ser

**Reply by AmaralSza** [reply]
Sim, funcionou certinho, estava testando o funcionamento dos grupos, ai ele parou de receber as mensagens do grupo, foi então que eu resolvi atualizar, na opção 4... E o mesmo agora só fica no erro 502, porém no navegador que eu já estava logado ele segue funcionando, quando eu abro um privado ele não abre nem a tela de login.

**Comment by cleitonme** [comment]
se mexeu algum arquivo da pasta instalada.. com usuario root da problemas de permisao dae tens que ver permissoes da pasta

**Reply by AmaralSza** [reply]
Estão todos como deploy deploy

**Reply by AmaralSza** [reply]
O estranho é que onde eu estava logado ainda segue funcionando, apenas se eu abrir em um privado ou outro navegador retorna o erro

**Comment by cleitonme** [comment]
que deu algum erro... na atualiacao tah no cache arquivos antigos, executa atualizacao denovo ou ve manual como fazer manualmente da verificada em grupos do izing

**Reply by AmaralSza** [reply]
Pois eh, já rodei a atualização outras vezes, reiniciei a maquina e tudo, mas não rolou, o estranho é que estava funcionando e depois da atualização deu isso, parece algo de errado com o atualizador

**Comment by cleitonme** [comment]
instalador nao tem nada errado, verifique em grupos, se alguem pode lhe ajudar.. ou se caso queira minha consultoria chamar no whatsapp (sera cobrado por isso)
